dataactbroker/handlers/fileHandler.py

```diff
@@ -969,7 +969,7 @@ def build_file_map(self, request_params, file_type_list, response_dict, upload_f
             if file_name:
                 if not self.isLocal:
                     upload_name = "{}/{}".format(
-                        g.user.user_id,
+                        submission.submission_id,
```

Comment:
No, that's what the one-time script I'm working on will do (although if someone uploads a new file in an existing submission this SHOULD work as intended as well)